### ISSUE TITLE
Error in nextcloud log

### ISSUE THREAD

**Issue by Loki3000** [nextcloud/issuetemplate#16]
```
Illegal offset type at /var/www/nextcloud/apps/issuetemplate/lib/Settings/Admin.php#220
```
220 string is
```
$repos[$appTitle] = $appinfo['bugs'];
```
I check $appTitle:
```
var_dump($appTitle);
array(2) {
  [0]=>
  string(5) "Notes"
  [1]=>
  array(2) {
    ["@attributes"]=>
    array(1) {
      ["lang"]=>
      string(2) "de"
    }
    ["@value"]=>
    string(7) "Notizen"
  }
}
```

is $appTitle really can be an array or it's error of Notes app?

**Comment by juliushaertl** [comment]
Hm that looks strange. $appTitle should be a string containing ` \OC_App::getAppInfo($app)['name']`. I'll try to reproduce this. What versions of Nextcloud/notes/issuetemplate are you using?

**Comment by Loki3000** [comment]
Nextcloud 11.0.2
Notes 2.2.0
Issue Template 0.2.1

**Comment by juliushaertl** [comment]
@Loki3000 Thanks for reporting. A fix for this issue will be in when #19 is ready.

**Comment by juliushaertl** [comment]
Fixed with #19
